libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/Action.kt

```diff
@@ -0,0 +1,3 @@
+package com.mapbox.navigation.dropin
+
+interface Action
```

Comment:
We can remove this file for now.

Reply:
This came from your branch. You want me to delete it?

Reply:
Yes. I think I removed it when merged to main.